[DO NOT MERGE]FRW-1441 Dynamic multi store integration guides (#1650)
spryker/spryker-docs: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md

Follow the steps below to integrate the Glue API: Dynamic Store feature into your project.

## Prerequisites
To start feature integration, overview and install the necessary features:

| NAME | VERSION |  
| --- | --- | --- |
| Spryker Core | {{page.version}} |

> ### 1) Set up behavior

Reviewer: ```suggestion
### Set up behavior
```

Send a request to the Glue API with the following headers:

```bash
Store: DE
```

Make sure that the store and locale are set correctly.

{% endinfo_block %}

**src/Pyz/Glue/GlueBackendApiApplication/GlueBackendApiApplicationDependencyProvider.php**

```php
<?php

namespace Pyz\Glue\GlueBackendApiApplication;

use Spryker\Glue\GlueBackendApiApplication\GlueBackendApiApplicationDependencyProvider as SprykerGlueBackendApiApplicationDependencyProvider;
use Spryker\Glue\StoresRestApi\Plugin\Application\StoreHttpHeaderApplicationPlugin;
 

class GlueBackendApiApplicationDependencyProvider extends SprykerGlueBackendApiApplicationDependencyProvider
{
    /**
     * @return array<\Spryker\Shared\ApplicationExtension\Dependency\Plugin\ApplicationPluginInterface>
     */
    protected function getApplicationPlugins(): array
    {
        return [
            new StoreHttpHeaderApplicationPlugin(),
        ];
    }
}
```

{% info_block warningBox "Verification" %}

If everything is set up correctly, a request to `https://glue-backend.mysprykershop.com` with the header `[{"key":"Store","value":"DE"}]` should result in a response without any errors.

> Example for test: 

Reviewer: ```suggestion
An example for testing: 
```

--data-urlencode 'grantType=password' \
--data-urlencode 'username={USERNAME}' \
--data-urlencode 'password={PASSWORD}'

```

{% endinfo_block %}

**src/Pyz/Glue/GlueStorefrontApiApplication/GlueStorefrontApiApplicationDependencyProvider.php**

```php
<?php

namespace Pyz\Glue\GlueStorefrontApiApplication;

use Spryker\Glue\GlueStorefrontApiApplication\GlueStorefrontApiApplicationDependencyProvider as SprykerGlueStorefrontApiApplicationDependencyProvider;
use Spryker\Glue\Http\Plugin\Application\HttpApplicationPlugin;
use Spryker\Glue\Locale\Plugin\Application\LocaleApplicationPlugin;
use Spryker\Glue\StoresRestApi\Plugin\Application\StoreHttpHeaderApplicationPlugin;

class GlueStorefrontApiApplicationDependencyProvider extends SprykerGlueStorefrontApiApplicationDependencyProvider
{
    /**
     * @return array<\Spryker\Shared\ApplicationExtension\Dependency\Plugin\ApplicationPluginInterface>
     */
    protected function getApplicationPlugins(): array
    {
        return [
            new HttpApplicationPlugin(),
            new StoreHttpHeaderApplicationPlugin(),
            new LocaleApplicationPlugin(),
        ];
}

```

{% info_block warningBox "Verification" %}

> If everything is set up correctly, a request to `https://glue-storefront.mysprykershop.com` with the header `[{"key":"accept-language","value":"de_DE"},{"key":"Store","value":"DE"}]` should result in a response whithout any errors and contains the `content-language` header set to **de_DE**.

Reviewer: ```suggestion
If everything is set up correctly, a request to `https://glue-storefront.mysprykershop.com` with the header `[{"key":"accept-language","value":"de_DE"},{"key":"Store","value":"DE"}]` should result in a response without any errors and will contain the `content-language` header set to **de_DE**.
```
